docs/how-to/websites-on-ipfs/redirects-file-support.md

```diff
@@ -0,0 +1,96 @@
+---
+title: redirects file support
+description: What the _redirect file is and how to use them with a website on IPFS.
+---
+# `_redirects` file support
+
+The `_redirects` file provides support for URL redirects and rewrites for websites hosted on Subdomain or DNSLink gateways. This feature enables support for single-page applications, progressive web applications, custom 404 pages, and avoids link rot when moving to IPFS-backed hosting. As well as the ability to change the appearance of a URL, change where content is located without breaking existing links, and enable URL rewriting.
```

Comment:
> hosted on Subdomain

Should _Subdomain_ be lowercase here? Or is it the name of a project or something? If so, can we link to it?

> As well as the

Using this kind of opener only really works when you're continuing a sentence. But saying that, the previous sentence is already pretty long. Try using `On top of that,` instead.